src/types.ts

```diff
@@ -11,8 +11,9 @@ export interface Component {
   /** @deprecated */
   global: boolean
   level: number
-  prefetch: boolean
-  preload: boolean
+  prefetch: boolean | number
```

Comment:
Are prefect and preload still required props?

Reply:
@clarkdo 

Yes, it's fine to set them as required keys of Component type, because default values are set in case they are `undefined`:

https://github.com/nuxt/components/blob/f8667b60d8e7e30188037004f8a4cbafca4c8a65/src/scan.ts#L20

In contrast, they are not required in ScanDir type, which is used to pass options to Nuxt Component module.